compiler/lib/driver.ml

```diff
@@ -397,6 +398,35 @@ let pack ~global { Linker.runtime_code = js; always_required_codes } =
   in
   let runtime_js = wrap_in_iifa ~can_use_strict:true js in
   let js = List.flatten always_required_js @ runtime_js in
+  let js =
+    match global, standalone with
+    | (`Function | `Bind_to _ | `Custom _), _ -> js
+    | `globalThis, false -> js
```

Comment:
What is `standalone` and why there's no need to stub `globalThis` when it's `false`?

Reply:
Standalone is false when building individual cmo/cma to javascript during separate compilaton. The globalThis compatibility layer will already be provided by the jsoo runtime loaded first.